### PR TITLE
issues

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
closes #2744

**Comment by SansPapyrus683** [comment]
@jjliewie how about you review mine this time

File: solutions/usaco-736.mdx [hunk 1]
```diff
@@ -76,33 +76,29 @@ public class BovineGeonomics {
...
 }
 ```
-</JavaSection>
 
+</JavaSection>
 <PySection>
 
 ```py
-# Take in the input data using python standard file I/O system
 file_in = open('cownomics.in')
```
Comment (jjliewie) [inline solutions/usaco-736.mdx:88]:
```suggestion
with open('cownomics.in') as read:
```

File: solutions/usaco-736.mdx [hunk 2]
```diff
@@ -76,33 +76,29 @@ public class BovineGeonomics {
...
-spotty_cows = []
-plain_cows = []
-
 spotty_cows = [data[i + 1] for i in range(n)]
 plain_cows = [data[i + n + 1] for i in range(n)]
 
-spots = []
-plains = []
-
 total = 0
```
Comment (jjliewie) [inline solutions/usaco-736.mdx:96]: maybe ans instead? or positions

File: solutions/usaco-736.mdx [hunk 3]
```diff
@@ -76,33 +76,29 @@ public class BovineGeonomics {
...
 
+</JavaSection>
 <PySection>
 
 ```py
-# Take in the input data using python standard file I/O system
 file_in = open('cownomics.in')
 data = file_in.read().strip().split('\n')
 
 n, m = map(int, data[0].split(' '))
```
Comment (jjliewie) [inline solutions/usaco-736.mdx:91]: no need to have ' '

```suggestion
n, m = map(int, data[0].split())
```

File: solutions/usaco-736.mdx [hunk 4]
```diff
@@ -76,33 +76,29 @@ public class BovineGeonomics {
...
 ```
-</JavaSection>
 
+</JavaSection>
 <PySection>
 
 ```py
-# Take in the input data using python standard file I/O system
 file_in = open('cownomics.in')
 data = file_in.read().strip().split('\n')
```
Comment (jjliewie) [inline solutions/usaco-736.mdx:89]:
```suggestion
        data = read.readline().strip().split('\n')
```

**Comment by jjliewie** [comment]
> @jjliewie how about you review mine this time

how the turn tables

File: solutions/usaco-736.mdx [hunk 5]
```diff
@@ -76,33 +76,29 @@ public class BovineGeonomics {
...
 <PySection>
 
 ```py
-# Take in the input data using python standard file I/O system
-file_in = open('cownomics.in')
-data = file_in.read().strip().split('\n')
-
-n, m = map(int, data[0].split(' '))
+with open('cownomics.in') as read:
+        data = read.readline().strip().split('\n')
```
Comment (ryanchou-dev) [inline solutions/usaco-736.mdx:89]: indent with tabs

(omg it happened again)

**Review by ryanchou-dev — APPROVED** [review]
other than this, gj!

**Review by jjliewie — APPROVED** [review]

**Pull request merged**